### DISCUSSION TITLE
Error Message: Runner is not Alive

### DISCUSSION THREAD

**Discussion by logyourtesting** [getgauge/gauge#2188] [General]
when running scenario from visual studio code getting this error :

**Logs**
Error Message: error reading from server: read tcp 444.0.0.1:49719->127.0.0.1:444718: wsarecv: An existing connection was forcibly closed by the remote host.
  Stacktrace: 
  
  Error Message: Runner is not Alive
  Stacktrace: 

**plugins and gauge versions**
Gauge version: 1.4.2
Commit Hash: f080fbc

Plugins
-------
html-report (4.1.4)
js (2.4.0)
screenshot (0.1.0)

**Comment by sriv** [comment]
looks like the runner crashed unexpectedly, is there any information in the logs?

**Comment by AleksaIT** [comment]
In my case the problem with this error was the bad import for step that AI gave me. After switching it as in the original documentation it worked:
import requests
from getgauge.python import step

You can find solid info in the logs.
